mesop/components/uploader/uploader.ng.html

```diff
@@ -0,0 +1,5 @@
+<input
```

Comment:
One thing we need to watch out for is that it's actually [quite difficult to style](https://blog.angular-university.io/angular-file-upload/#:~:text=Why%20don%27t%20we%20see%20file%20inputs%20very%20often%3F) the native file input element.

Can we use the angular material button as the base UI for this? It seems like what most people would want (and a button is fairly easy for app developers to customize).

Reply:
That's a good point. I've definitely done the hidden file input trick before. Thanks for the link for how to do it in Angular.

I ended up using the angular button. But I had some questions about this:

1. Should I use the mesop button instead (if so how would I get it imported?)
2. I didn't expose any styling for the button yet. Mainly, I think we can probably improve our uploader with some more features like drag/drop, multiple uploads, and such, which I think would change the look of the component. For now I just kept it as basic as possible.

Reply:
LGTM. I think using the Angular button is fine here. SG with keeping the styling simple, we could expose the style attribute later, but something basic is a good start.